README.rst

```diff
@@ -1107,3 +1107,87 @@ And more...
 -----------
 
 Additional code samples are available in the `test directory <https://github.com/irods/python-irodsclient/tree/master/irods/test>`_
+
+
+=======
+Testing
+=======
+
+Setting up and running tests
+----------------------------
+
+The Python iRODS Client comes with its own suite of tests.  Some amount of setting up may be necessary first:
+
+1. Use :code:`iinit` to specify the iRODS client environment.
+   For best results, point the client at a server running on the local host.
+
+2. Install the python-irodsclient along with the :code:`unittest unittest_xml_reporting` module or the older :code:`xmlrunner` equivalent.
+
+   - for PRC versions 1.1.1 and later:
+
+     *  :code:`pip install ./path-to-python-irodsclient-repo[tests]`  (when using a local Git repo); or,
+     *  :code:`pip install python-irodsclient[tests]'>=1.1.1'`  (when installing directly from PyPI).
+
+   - earlier releases (<= 1.1.0) will install the outdated :code:`xmlrunner` module automatically
+
+3. Follow further instructions in the `test directory <https://github.com/irods/python-irodsclient/tree/master/irods/test>`_
```

Comment:
master -> main

Reply:
changed both links that were pointing to a .../master/... link to  .../main/... instead.